.github/workflows/ci.yml

```diff
@@ -64,6 +70,10 @@ jobs:
         if: ${{ steps.cache-node-modules.outputs.cache-hit != 'true' }}
         run: npm ci
 
+      - name: NX Reset Cache
+        if: ${{ env.NX_RESET_CACHE == 'true' }}
+        run: npx nx reset
```

Comment:
as per docs, this command will also stop the nx deamon  so we minght need something different